src/Microsoft.Health.Fhir.ValueSets/CurrencyValues.cs

```diff
@@ -0,0 +1,12 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.ValueSets
+{
+    public static class CurrencyValues
```

Comment:
I'm wondering if these names should be flipped so that this file could be reused if needed. Something like the class being SystemValues and the const being Currency. As it is I don't know what we could use this for outside this PR.

Reply:
There are a lot of valuesets http://www.hl7.org/implement/standards/fhir/terminologies-valuesets.html.
The difference here is that most things in assembly are defining the ValueSet values, whereas this is the valueset system

Reply:
I think I was thrown with `SystemValues` :P There is the valueset system and the valueset values. SystemValues made me think twice

Reply:
Good point! I originally decided on this way to parallel the implementation of [SpecialValues](https://github.com/microsoft/fhir-server/blob/master/src/Microsoft.Health.Fhir.ValueSets/SpecialValues.cs). I think you're right though - it would be nice to put these in one file.

I'll wait to see what feedback I get on [this comment](https://github.com/microsoft/fhir-server/pull/1475#discussion_r527322562) before committing a change (I want to confirm we actually need this hard coded string).

Reply:
@brendankowitz sorry, I didn't see your comment before adding my reply (GitHub has been weird today). By `SystemValues` do you mean `SpecialValues.System`?

Reply:
Perhaps something like CurrencyValueSet.System is fine.

Reply:
Good idea, thanks. Renamed in 02c118d.

Summarizing our earlier discussion: the `ValueSet` uri is "http://hl7.org/fhir/ValueSet/currencies" and the `CodeSystem` is "urn:iso:std:iso:4217", so it would be good to avoid "SystemValues" ([here](https://www.hl7.org/fhir/valueset-currencies.html) is the relevant part of the spec).